xbmc/favourites/FavouritesUtils.cpp

```diff
@@ -127,11 +127,11 @@ bool ShouldEnableMoveItems()
 
 namespace
 {
-bool ExecuteAction(const std::string& execString)
+bool ExecuteAction(const std::string& execString, const std::shared_ptr<CFileItem>& item)
```

Comment:
In the past I fixed a bug due to passing a std::shared_ptr<CFileItem> reference around, and the caller dropped the refcount to zero and rugpulled the pointer. Is it possible to pass by value, forcing a copy, and then do the best to elide a copy later with std::move()?

Reply:
> and the caller dropped the refcount to zero and rugpulled the pointer.

I see your point, here it is most likely not happening, because the item is only know to a single thread.

Reply:
Sounds good, just wanted to scrutinize that